Examples/KerrBH4dST/CouplingAndPotential.hpp

```diff
@@ -51,7 +51,7 @@ class CouplingAndPotential
                  exp(-m_params.quadratic_factor * vars.phi * vars.phi *
                      (1. + m_params.quartic_factor * vars.phi * vars.phi)) *
                  vars.phi *
-                 (1. + 2. * m_params.quadratic_factor * vars.phi * vars.phi);
+                 (1. + 2. * m_params.quartic_factor * vars.phi * vars.phi);
```

Comment:
I guess this was a typo originally?

Reply:
Yes it was! I figured it out when the scalarisation of Kerr was giving me different results